scaling-rules.json

```diff
@@ -0,0 +1,36 @@
+{
+    "instance_min_count": 1,
+    "instance_max_count": 5,
+    "scaling_rules": [
+        {
+            "metric_type": "memoryused",
+            "threshold": 100,
```

Comment:
I think the threshold 100M for memory is not  proper because the default quota is 512M(or 1024M) and the scale out is easily triggered by a little amount of requests, the result is that we can't take fully usage of memory quota.